app/components/primer/alpha/action_bar.pcss

```
@@ -0,0 +1,69 @@
+/* CSS for ActionBar  */
+.ActionBar {
+  position: relative;
+  display: flex !important;
+  min-width: calc(var(--primer-control-medium-size, 32px) * 3);
```

```suggestion
  min-width: calc(var(--control-medium-size) * 3); 
```

This took me awhile to understand. Maybe add a comment like "show minimum of 3 buttons"